set.mm

```diff
@@ -7643,7 +7652,7 @@ inference eliminates the need for a syllogism ( ~ syl ) -type inference
   1. Mixed Op means in this section, that both df-or and df-an is referenced in
   a theorem and its proof.  The first part is mostly populated with theorems
   that are in principle derivable from df-or only.  They have only ` \/ ` in
-  both the hypotheses and conclusion, but their proof benefit from shortcuts
+  both hypotheses and conclusion, but their proof benefit from shortcuts
```

Comment:
Add an "s" to either "proof" or "benefit" (both spellings are used and both are ambiguous...).